CanvasPlusPlayground/Features/People/PeopleView.swift

```diff
@@ -81,33 +72,71 @@ struct PeopleView: View {
             suggestedTokens: .constant(suggestedTokens),
             prompt: "Search People..."
         ) { token in
-            Label(token.text, systemImage: "person.fill")
+            Label(token.category.displayName, systemImage: "person.fill")
         }
         #endif
         .overlay {
-            if !searchText.isEmpty && displayedUsers.isEmpty {
+            noResultsBanner
+        }
+        .onChange(of: searchText) { _, _ in
+            newQueryAsync()
+        }
+        .onChange(of: selectedTokens) { _, _ in
+            newQueryAsync()
+        }
+    }
+
+    @ViewBuilder
+    var noResultsBanner: some View {
+        if peopleManager.loadingState != .loading {
+            if !searchText.isEmpty && peopleManager.displayedUsers.isEmpty {
                 ContentUnavailableView("No results for '\(searchText)'", systemImage: "magnifyingglass")
```

Comment:
While you're here can you update this to be:
```suggestion
                ContentUnavailableView.search(text: searchText)
```

Reply:
Oh ok, didn't know there was a search default.